src/redux/error/error.selectors.test.js

```diff
@@ -0,0 +1,17 @@
+import { getErrorMessage } from './selectors'
+
+describe('selectors/getErrorMessage', () => {
+
+  describe('#getErrorMessage(state)', () => {
+    it('returns message when state has an error', () => {
+      const state = { error: { response: { message: 'foo'} } }
+      expect(getErrorMessage(state)).to.equal('foo')
+    })
+
+    it('returns undefined when state does not have an error', () => {
+      const state = { }
+      expect(getErrorMessage(state)).to.be.undefined
+    })
+  })
```

Comment:
sorta seems like parsing an error message is going to be the same, but not sure if that means it should have 1 selector or what